charts/ceph-csi-rbd/values.yaml

```diff
@@ -300,6 +300,9 @@ provisioner:
   podSecurityContext: {}
 
 topology:
+  # Specifies whether topology based provisioning support should
+  # be exposed by CSI
+  enabled: false
```

Comment:
this option needs to be added to the documentation too

Reply:
Actually it is available already (I had forgot to remove it in my previous PR 😅).
https://github.com/ceph/ceph-csi/blob/a3d457a8dd010eb32ef9427eebc33e1357ec45f9/charts/ceph-csi-rbd/README.md?plain=1#L180-L181